### DISCUSSION TITLE
Troubles with Tasmota-Bluetooth Firmware 13.4.1.2

### DISCUSSION THREAD

**Discussion by wjohn007** [arendst/Tasmota#21328] [General]
I used following link weeks ago to install Tasmota firmware on two devices.

https://github.com/tasmota/install/raw/firmware/firmware/unofficial/tasmota32solo1-bluetooth.bin

Everything worked fine. The MI-Temperature sensors were detected as expected.
Firmware: 13.4.0.2 (5e8b116-solo1)

I recently updated a device with Tasmota 13.4.1.2 (f7e285a-solo1).
This device does not detect a BLE sensor.

Is there a solution or explanation for this ?

**Comment by Jason2866** [comment]
Tasmota Arduino Core is changed from 2.0.15 to 3.0.0. The BT driver has been adopted to compile. It is not tested a lot so it is possible it is not working.
You can use the alternative MI BT build for MI sensors.

**Comment by wjohn007** [comment]
I suspected that. Thanks for the hint.

**Comment by Rushmed** [comment]
Will there be a solution for percompiled tasmota32-bluetooth.bin?
Do we need an issue to keep on that?

**Comment by Mark-Muc** [comment]
Hello ..seems that it also not working on 14.0.0.1 dev if u compile our own version!
Regnards
Mark_Muc

**Comment by wjohn007** [comment]
I created my own tasmota32solo1-MI32.bin based on Tasmota 14.0.0. It works as expected.
@Jason2866
Unfortunately, there is no unofficial version of this variant. I would like to see tasmota32solo1-MI32.bin added to the list of unofficial images.

**Reply by Jason2866** [reply]
why? The Bluetooth bug will be fixed. There are enough other precompiled MI variants.
EDIT: Issue is fixed in branch development.

**Reply by wjohn007** [reply]
@Jason2866 
Thank you very much for bug fixing
